File: New folder/notes.erl
% erlang notes
-module(notes).
-export([greet/1, main/0]).

greet(Name) ->
    "hello " ++ Name.

main() ->
    io:format("~s~n", [greet("practice")]),
    Nums = [1, 2, 3],
    Doubled = lists:map(fun(N) -> N * 2 end, Nums),
    io:format("~p~n", [Doubled]).

% TODO: try processes/messages next

File: New folder/notes.rs
// rust notes
fn greet(name: &str) -> String {
    format!("hello {}", name)
}

fn main() {
    println!("{}", greet("practice"));

    let nums = vec![1, 2, 3];
    for n in &nums {
        println!("{}", n * 2);
    }
}

// TODO: try ownership/borrowing deeper next
